metadata/boundaries/zip-code-areas

```diff
@@ -0,0 +1,83 @@
+
+## Brief Summary
+
+The estimated geographic extents for all ZIP codes in Utah, excluding those that are PO Box only.
+
+## Summary
+
+This dataset includes the approximate geographic boundaries of every five-digit Zone Improvement Program (ZIP) code in Utah. ZIP codes are approximated based on the locations of known mailing addresses and are represented as polygons.
+
+## Description
+
+### What is the dataset?
+
+ZIP codes were first introduced by the United State Postal Service (USPS) in 1963 to improve mail delivery. The first three digits of the five-digit code refer to the Sectional Center Facility (SCF), or the central processing facility for a given region's mail. The last two digits provide a more specific locale that the mail will be delivered to.
+
+### What is the purpose of the dataset?
+
+This dataset was developed to provide a reference for assigning ZIP codes to features in the SGID such as [Utah Roads](https://gis.utah.gov/products/sgid/transportation/road-centerlines/) and [Utah Address Points](https://gis.utah.gov/products/sgid/location/address-points/) datasets. Utah ZIP Code Areas have also been used in a variety of analyses; for example, one such analysis maps housing affordability in Utah by ZIP code. You can read about how ZIP codes were used to quantify housing data on our [blog](https://gis.utah.gov/blog/2024-05-17-utah-housing-affordability-map/).
+
+This dataset should not be used to determine the definitive ZIP code for addressing mail that will be sent through the US Postal Service. The [USPS website](https://tools.usps.com/zip-code-lookup.htm) should be used for this purpose.
+
+### What does the dataset represent?
+
+Each polygon in this dataset represents the approximate geographic area of a given ZIP code and includes information such as the city and county the code lies within. These polygons were derived from a database of road polylines originally used to improve mail delivery for carriers. UGRC has translated that polyline data into polygons for visualization and analysis purposes.
+
+### How was the dataset created?
+
+UGRC produced this dataset from multiple data sources using a combination of methods. Editing methods included:
+
+- Using local ZIP code polygon data obtained from counties and cities.
+- Identifying place locations (cities, towns, populated areas, etc.) from the postal service website and address information system (AIS) and building ZIP code boundaries around these places.
+- Using the 2000 Census TIGER line file’s ZIP code attributes.
+
+<!--- I've got a list of updates for this dataset, but have been unsuccessful in finding how this dataset is updated or what needs to happen in order to justify an update. Will be gathering more information on this. -->
+
+### How reliable and accurate is the dataset?
+
+This dataset represents approximate boundaries only. Please reach out to [our team](https://gis.utah.gov/contact/) with questions concerning this dataset.
```

Comment:
Should we add something that says "We discourage the usage of this data layer unless necessary as the accuracy is not guaranteed."?

Reply:
My thought with this is we don't necessarily need to discourage people from using the layer, so much as to point out what it can and cannot be used for. I've since updated this section, it now reads:

> This dataset represents approximate boundaries only and serves as a reference for cartographic and general analytic purposes. This dataset should not be used to determine the definitive ZIP code for addressing mail that will be sent through the US Postal Service. The [USPS website](https://tools.usps.com/zip-code-lookup.htm) should be used for this purpose.

Is there anything else that you think we should specify it cannot be used for? I do agree a bit of disclaimer goes a long way.

Reply:
This is great. Thank you

Reply:
i think that wording looks great